src/pipecat/transports/network/webrtc_connection.py

```diff
@@ -94,14 +95,16 @@ class SmallWebRTCTrack:
     enable/disable control and frame discarding for audio and video streams.
     """
 
-    def __init__(self, track: MediaStreamTrack):
+    def __init__(self, track: MediaStreamTrack, index: int):
         """Initialize the WebRTC track wrapper.
 
         Args:
             track: The underlying MediaStreamTrack to wrap.
+            index: The index of the track in the transceiver (0 for mic, 1 for cam, 2 for screen)
         """
         self._track = track
         self._enabled = True
+        self.source_index = index
```

Comment:
Are we using this `self.source_index` somewhere ?

Reply:
i bet i was using it for debug logging and forgot to remove it when i removed the logs. It _is_ sort of handy for that, but i'll remove it.